firebase-functions/README.md

```diff
@@ -0,0 +1,2 @@
+# Medium Post
+https://medium.com/@kmltrk07/how-to-deploy-gofiber-app-to-firebase-functions-8d4d537a4464
```

Comment:
can you add a bit more, a short description of the example

please also link in the root readme